src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    public static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
Follow the encapsulation principle. It's a bad practice to make your class fields public or static. Consider using private access modifier and providing access through methods.